src/v/kafka/requests/fetch_request.cc

```diff
@@ -332,6 +332,16 @@ static ss::future<read_result> read_from_partition(
       });
```

Comment:
please use the correct syntax when referring to github issues in the commit message.

Reply:
nice catch was missing the number sign